tasks/__init__.py

```diff
@@ -520,6 +520,7 @@ def test(ctx, all=False, syntax=False):
         flake(ctx)
         jshint(ctx)
 
+    test_js(ctx)
```

Comment:
I believe we are currently maxed out on branches, so added to our fastest, please.